sheet.go

```diff
@@ -110,6 +110,12 @@ func (s *Sheet) makeXLSXSheet(refTable *RefTable, styles *xlsxStyleSheet) *xlsxW
 				xFont, xFill, xBorder, xCellStyleXf, xCellXf := style.makeXLSXStyleElements()
 				fontId := styles.addFont(xFont)
 				fillId := styles.addFill(xFill)
+				// generate NumFmtId
+				xNumFmt := styles.newNumFmt(cell.numFmt)
+				// the NumFmtId of NumFmt user defined is greater than 164
+				if xNumFmt.NumFmtId >= 164 {
```

Comment:
The number format id less than 164 is builtin. 
d4320de787cb2e14773b63537a50ea865e585c1d fixed this.